src/components/scenes/ConfirmScene.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react'
+import { View } from 'react-native'
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
+
+import { useHandler } from '../../hooks/useHandler'
+import s from '../../locales/strings'
+import { NavigationProp, RouteProp } from '../../types/routerTypes'
+import { SceneWrapper } from '../common/SceneWrapper'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+import { MainButton } from '../themed/MainButton'
+import { SafeSlider } from '../themed/SafeSlider'
+import { SceneHeader } from '../themed/SceneHeader'
+import { Tile } from '../tiles/Tile'
+interface Props {
+  navigation: NavigationProp<'confirmScene'>
+  route: RouteProp<'confirmScene'>
+}
+
+export interface ConfirmSceneParams {
+  titleText: string
+  bodyText: string
+  infoTiles?: Array<{ label: string; value: string }>
+  onConfirm: (resetSlider: () => void) => void
```

Comment:
Optional: I think this callback should be `async`. In the old send scene with it's insane redux logic, we needed the `resetSlider` solution to indicate that `onConfirm` had completed, and the slider could be released. Since this is fresh code, we can just reset the slider when the promise resolves.

This is the pattern we use for `MainButton`, `ButtonsModal`, and a bunch of other stuff in the GUI, and it works pretty well when you are just doing something simple.